RELEASE-NOTES.txt

```diff
@@ -16,6 +16,8 @@
 * Added user Gravatar to the Me menu in My Site.
 * Updated site details screen title to "My site", to avoid duplicating the title of the current site which is displayed in the screen's header area. 
 
+* @-mention suggestions are now presented immediately after typing "@"
```

Comment:
Maybe change from "typing" to "entering" or "pressing" "@" since the ability to type "@" on the keyboard is a design request that is not yet implemented.

Reply:
Good catch, I think you're right! I actually need to remove this entirely for now though because we're going to keep this under a dev flag for the time being.